packages/cli/package.json

```diff
@@ -60,6 +59,9 @@
         "@types/npm-package-arg": "^6.1.1",
         "@types/promptly": "^3.0.2",
         "babel-loader": "^9.1.2",
+        "esbuild": "^0.17.19",
+        "ts-node": "^10.9.1",
+        "typescript": "^5.0.4",
```

Comment:
unrelated as well. moving compiler and builder out of the dependencies to the devDep